hazelcast/src/test/java/com/hazelcast/internal/nearcache/AbstractNearCacheBasicTest.java

```diff
@@ -335,7 +335,8 @@ public void testGetAsyncPopulatesNearCache() throws Exception {
 
         // generate Near Cache hits
         populateNearCacheAsync(context);
-
+        assertEquals("Not equals to expected Near Cache hit count",
```

Comment:
If not then you can use `context.stats` instead of `context.nearCache.getNearCacheStats()`